src/components/WebexSettings/Tabs.jsx

```diff
@@ -97,6 +97,7 @@ Tabs.propTypes = {
   onSelect: PropTypes.func.isRequired,
   selected: PropTypes.string.isRequired,
   style: PropTypes.shape(),
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
Why is this eslint disable needed?

Reply:
While committing was observing error for these lines

Reply:
So shouldn't we fix the root cause instead of blocking the eslint error? @Kesari3008

Reply:
removed it